docs/migrations/rest.md

```diff
@@ -32,28 +32,28 @@ Some important information concerning all legacy REST endpoints:
+| `GET /bank/balances/{address}`                                                  | Get the balance of an address                                       | `GET /cosmos/bank/v1beta1/balances/{address}`                                                 |
 | `GET /bank/total`                                                               | Get the total supply of all coins                                   | `GET /cosmos/bank/v1beta1/supply`                                                                     |
 | `GET /bank/total/{denom}`                                                       | Get the total supply of one coin                                    | `GET /cosmos/bank/v1beta1/supply/{denom}`                                                             |
 | `POST /distribution/delegators/{delegatorAddr}/rewards`                         | Withdraw all delegator rewards                                      | N/A, use Protobuf directly                                                                            |
 | `POST /distribution/*`                                                          | Create unsigned `Msg`s for distribution                             | N/A, use Protobuf directly                                                                            |
 | `GET /distribution/delegators/{delegatorAddr}/rewards`                          | Get the total rewards balance from all delegations                  | `GET /cosmos/distribution/v1beta1/v1beta1/delegators/{delegator_address}/rewards`                     |
 | `GET /distribution/delegators/{delegatorAddr}/rewards/{validatorAddr}`          | Query a delegation reward                                           | `GET /cosmos/distribution/v1beta1/delegators/{delegatorAddr}/rewards/{validatorAddr}`                 |
 | `GET /distribution/delegators/{delegatorAddr}/withdraw_address`                 | Get the rewards withdrawal address                                  | `GET /cosmos/distribution/v1beta1/delegators/{delegatorAddr}/withdraw_address`                        |
-| `GET /distribution/validators/{validatorAddr}`                                  | Validator distribution information                                  | `GET /cosmos/distribution/v1beta1/validators/{validatorAddr}`                                         |
-| `GET /distribution/validators/{validatorAddr}/rewards`                          | Commission and self-delegation rewards of a single a validator      | `GET /cosmos/distribution/v1beta1/validators/{validatorAddr}/rewards`                                 |
+| `GET /distribution/validators/{validatorAddr}`                                  | Validator distribution information                                  | N/A                                         |
+| `GET /distribution/validators/{validatorAddr}/rewards`                          | Commission and self-delegation rewards of a single a validator      | N/A                                 |
```

Comment:
Could that be replaced with `ValidatorOutstandingRewards` and `ValidatorCommission`?